solution/backend/serverless-experimental.yml

```diff
@@ -91,7 +91,7 @@ functions:
   drop_database:
     environment:
       STAGE: ${self:custom.stage}
-      DB_NAME: main
+      DB_NAME: eregs
```

Comment:
is this the same DB_NAME as the one defined on line 12?

Reply:
no.  And we dont want it to be.  If you connect to the db instance using stage, so the pr name, and you drop it, it causes an error for terminating connection. So you need to connect to a database that is not the same as the one you are dropping.